ft_freqanalysis.m

```diff
@@ -106,10 +106,18 @@
 %   cfg.foilim     = [begin end], frequency band of interest
 %   cfg.toi        = vector 1 x numtoi, the times on which the analysis
 %                    windows should be centered (in seconds)
-%   cfg.width      = 'width', or number of cycles, of the wavelet (default = 7)
+%   cfg.width      = 'width', or number of cycles, of the wavelet or base wavelet for 
+%                    superlets(default = 7)
 %   cfg.gwidth     = determines the length of the used wavelets in standard
 %                    deviations of the implicit Gaussian kernel and should
 %                    be choosen >= 3; (default = 3)
+%   cfg.superlets  = 'no', 'additive', 'multiplicative' (default = 'no')
```

Comment:
I would like to suggest to organize the code (and options) slightly differently.
Rather than having cfg.superlets = 'no', 'additive', 'multiplicative', I'd recommend to have 

cfg.method = 'superlet';

and to deal with this as a separate method in the switch-boxes below.

In other words, keep the handling of the 'normal'  wavelet as is, and handle the superlets in a separate case.

For the option-handling, I'd make a substructure (cfg.superlet) that handles the method specific options. Admittedly, this is not how ft_freqanalysis deals with the options of the methods (this has grown historically), but this is how less ancient (or more often updated) fieldtrip functions deal with it. For instance, cfg.<somemethod>.someparameter in ft_sourceanalysis.

So, you would need cfg.superlet.sl_order, and cfg.superlet.combine (or so) to be defined.